src/wp-includes/blocks/search.php

```diff
@@ -367,12 +367,12 @@ function styles_for_block_core_search( $attributes ) {
 	// Add color styles.
 	$has_text_color = ! empty( $attributes['style']['color']['text'] );
 	if ( $has_text_color ) {
-		$button_styles[] = sprintf( 'color: %s;', esc_attr( $attributes['style']['color']['text'] ) );
+		$button_styles[] = sprintf( 'color: %s;', $attributes['style']['color']['text'] );
 	}
 
 	$has_background_color = ! empty( $attributes['style']['color']['background'] );
 	if ( $has_background_color ) {
-		$button_styles[] = sprintf( 'background-color: %s;', esc_attr( $attributes['style']['color']['background'] ) );
+		$button_styles[] = sprintf( 'background-color: %s;', $attributes['style']['color']['background'] );
```

Comment:
Is it intended? Should we replace `esc_attr` with `esc_html` instead?

Reply:
Yeah, it's intended. We're not really removing the `esc_attr` but moving to the [latest possible point](https://github.com/WordPress/wordpress-develop/pull/3485/files#diff-b2e87a1e901a51c9ba26e81131bac3c18258a1c4310c21a276f2d98598d96c95R403). (So I don't think we should add `esc_html` here.)

FWIW, this change was part of yesterday's 6.0.3 (and earlier versions) security release.

Reply:
Alright, we're good to go then!